int8 ec_id          # ethercat板子的索引
int8 slave_id       # ethercat板子的子板的索引
int8 passage        # 通道号
int8 motor_id       # 电机id
int8 config_code    # 设置码
float32 param_a     # 第一个参数
float32 param_b     # 第二个参数
